Translators/Incident/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM python:3.10-slim
+
+ENV PYTHONUNBUFFERED=TRUE
+
+ENV WORKDIR=/app
+
+WORKDIR $WORKDIR
+
+COPY Translators/Incident/ .
```

Comment:
question: do we also need to copy over RoadConditions translator?

Reply:
We don't - all of the incident translator functionality is copied over, and each of these Dockerfiles corresponds to a single translator

Reply:
ahh I see. I was thinking it'd all be in one not separate but this makes more sense. thanks!